lib/reason_codes.yml

```diff
@@ -817,7 +817,7 @@ NOOR:
   probable_status: declined
 PART:
   code: PART
-  iso_name: "-"
+  iso_name: Parial Rejection
```

Comment:
Here s a typo

Reply:
Thanks! You can open a PR if you would like to.